component/helper/pom.xml

```diff
@@ -29,27 +29,41 @@
     <packaging>bundle</packaging>
     <name>WSO2 Carbon - Library For Identity Extension Helper</name>
     <url>http://wso2.org</url>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.ops4j.pax.logging</groupId>
+            <artifactId>pax-logging-api</artifactId>
+            <version>${pax.logging.api.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.wso2.carbon.identity.framework</groupId>
+            <artifactId>org.wso2.carbon.identity.application.common</artifactId>
+            <version>${carbon.identity.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.wso2.carbon.identity.framework</groupId>
+            <artifactId>org.wso2.carbon.identity.core</artifactId>
+            <version>${carbon.identity.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.wso2.carbon.identity.framework</groupId>
+            <artifactId>org.wso2.carbon.identity.application.authentication.framework</artifactId>
+            <version>${carbon.identity.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.wso2.carbon</groupId>
+            <artifactId>org.wso2.carbon.user.core</artifactId>
+            <version>${carbon.kernel.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.wso2.carbon.identity.framework</groupId>
+            <artifactId>org.wso2.carbon.identity.mgt</artifactId>
+            <version>${carbon.identity.version}</version>
+        </dependency>
+    </dependencies>
     <build>
-        <pluginManagement>
-            <plugins>
-                <plugin>
-                    <groupId>org.apache.felix</groupId>
-                    <artifactId>maven-bundle-plugin</artifactId>
-                    <version>2.3.5</version>
-                    <extensions>true</extensions>
-                </plugin>
-            </plugins>
-        </pluginManagement>
         <plugins>
-            <plugin>
-                <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-compiler-plugin</artifactId>
```

Comment:
why are we removing the plugins maven-bundle-plugin, maven-compiler-plugin and maven-scr-plugin

Reply:
maven-scr-plugin has been removed since we're moving away from legacy @scr.component annotations.

Reply:
maven-bundle-plugin is redundant [1], that's why an occurrence has been removed.

[1] https://github.com/wso2-extensions/identity-extension-utils/pull/51/files#diff-7461ac39fe82990d6b35d4289d42f837799cce92282691eb76e9c62446499c2cR69

Reply:
Added maven compiler plugin with https://github.com/wso2-extensions/identity-extension-utils/pull/51/commits/14de92d83770fc802ddf049551c74ab4ace675e5